docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -74,6 +74,7 @@ Property name                                             Required   Description
 ``pinot.aggregation-pushdown.enabled``                    No         Push down aggregation queries, default is ``true``.
 ``pinot.count-distinct-pushdown.enabled``                 No         Push down count distinct queries to Pinot, default is ``true``.
 ``pinot.target-segment-page-size``                        No         Max allowed page size for segment query, default is ``1MB``.
+``pinot.proxy.rest-uri``                                  No         Pinot Proxy Uri for controller/broker/server grpc requests, default is null.
```

Comment:
Why not just `pinot.proxy.uri`?

Also, capitalize URI in the description: "Pinot Proxy URI for ...."